bin/concatenate_all.py

```diff
@@ -33,6 +34,10 @@ def main():
         semester = lichen_config_data["semester"]
         course = lichen_config_data["course"]
         gradeable = lichen_config_data["gradeable"]
+        expressions = None
+        if("regrex" in lichen_config_data):
+        	#this assumes regex is seperated by a ','
+        	expressions = lichen_config_data["regrex"].split(',')
```

Comment:
`regrex` -> `regex`

Reply:
The configuration webpage dumps a json object with field `regrex` which I believe originally was a typo. I will make changes in one of my current PR's in the Submitty repo and update this as well.